docs/product_search/parameters.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: default
+title: Product Search Parameters
+parent: Product Search
+---
+
+# Product Search Parameters
+Product search is based on query parameters that determine which products are returned and in which order.
+
+### active_promotion
+Accepts: `true` or `false`.<br>
+Passing `true` will return items with a currently active [promotion]({% link docs/reference/objects/product/promotion.md %}#promotionactive).
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search active_promotion: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[active_promotion]=true
+```
+
+### category
+Accepts any of the Easol predefined categories.<br>
+Will accept either a single category or an array.<br>
+Will return products which match the [category]({% link docs/reference/objects/product/index.md %}#productcategory) passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search category: 'active' %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[category]=active
+```
+
+### country
+Accepts any of the Easol predefined [countries]({% link docs/reference/objects/product/index.md %}#productcountry) as either Country Codes or Country Names<br>
+Will accept either a single country or an array.<br>
+Will return products which match the country passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search country: 'FR' %}
+{% endproduct_search %}
+
+{% product_search country: ['FR','DE'] %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[country]=DE
+```
+
+### departure_date
+Accepts an object which specifies how to handle the search through `equal_to` `greater_than` `greater_or_equal_than` or `less_than` each taking a date in SQL format `YYYY-MM-DD`.
+Will return experiences which depart within the [departure date]({% link docs/reference/objects/product/index.md %}#productdepart_on) range.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search departure_date: {greater_or_equal_than: 'now', less_than: '2023-12-28' } %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as query paramameters
+```
+https://mysite.com/search?search[departure_date][greater_or_equal_than]=2022-11-19&search[departure_date][less_than]=2023-12-28
+```
+
+### departure_month
+Accepts a month as either a 3 letter abreviation, or the month number i.e. `Apr` or `4`.<br>
+Will return experiences which [depart]({% link docs/reference/objects/product/index.md %}#productdepart_on) within the month passed.
```

Comment:
'month passed' might sound like in the past. maybe  'within that month'  or 'within the specified month'
Worth also mentioning, if there are experiences across multiple years, it will return e.g. Apr 2023 and Apr 2024